make_deb

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+##--------------------------------------------------------------------
+## Copyright (c) 2018 Dianomic Systems
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##--------------------------------------------------------------------
+##
+## Author: Monika Sharma
+##
+
+set -e
+
+GIT_ROOT=`pwd`    # The script must be executed from the root git directory
+usage="$(basename "$0") [clean|cleanall]
+This script is used to create the Debian package of foglamp-display
+
+Arguments:
+ clean    - Remove all the old versions saved in format .XXXX
+ cleanall - Remove all the versions, including the last one"
+
+for i in "$@"
+do
+  case "$i" in
+      clean)
+          echo -n "Cleaning the build folder from older versions..."
+          find "${GIT_ROOT}/packages/build" -maxdepth 1 | grep '.*\.[0-9][0-9][0-9][0-9]' | xargs rm -rf
+          echo "Done."
+
+          exit 0
+          ;;
+      cleanall)
+          if [ -d "${GIT_ROOT}/packages/build" ]; then
+            echo -n "Cleaning the build folder..."
+            rm -rf ${GIT_ROOT}/packages/build/*
+            echo "Done."
+          else
+            echo "No build folder, skipping cleanall"
+          fi
+
+          exit 0
+          ;;
+      *)
+          echo "${usage}"
+          exit 1
+          ;;
+  esac
+done
+
+# OS Type
+os=$(uname)
+echo "Operating System Type: ${os}"
+
+if [[ ${os} == "Darwin" ]]
```

Comment:
Better to check if not `Linux`, this script is not compatible with Windows platform too

Reply:
Yes. 

```
if [[ ${os} != "Linux" ]]
then
  echo "This script is compatible with Linux only!"
  exit 1
fi
```

Reply:
Though we can build on mac, but that requires 2 things... **which we we should NOT add / support** 
1) brew install dpkg
2) `sed -i '' "s/Version: 0.0.0/Version: ${version}/g" DEBIAN/control` line 110 mind  extra `''` after -i